e2e-tests/cypress/e2e/Report-generation.cy.ts

```diff
@@ -0,0 +1,109 @@
+// report-generation.spec.ts
```

Comment:
when I run the single test in cypress UI I got: "Invalid Version: NOT_SET_YET". I recall you had this problem. Could you please tell us how to solve it?

Reply:
You need to use `$ cd e2e-tests; yarn cypress open --env kibanaVersion=[kbnVersion]`, for example `$ cd e2e-tests; yarn cypress open --env kibanaVersion=7.17.22`.

README.md file
> #### Cypress tests in interactive GUI
>
>```bash
>$ cd e2e-tests; yarn cypress open
>```

Reply:
I run it locally (8.14.3) and I got sth like this:
<img width="1155" alt="Screenshot 2024-09-02 at 18 40 46" src="https://github.com/user-attachments/assets/c51cb38d-a2d6-46a2-9de6-e1e2b90ce69f">

@Dzuming could you please check on your side?

Reply:
I've checked for KBN 7.17.22 and it works. So it seems the problem is related to KBN 8.x

Reply:
Yes, I can confirm. Seems that the test code should be adjusted to the Kibana 8.x -> We should change 8.4.3 to the 8.5.0 version since it's the newest supported version.

![Report Generation -- should generate and verify reports for user2, and user3 should not see them (failed)](https://github.com/user-attachments/assets/d1840baa-8406-4b9b-81c1-4649568a9c62)

![Report Generation -- should generate and verify reports for user2, and user3 should not see them (failed) (attempt 2)](https://github.com/user-attachments/assets/513ddb46-ccf3-44fa-9c70-92805a468c36)

![Report Generation -- should generate and verify reports for user2, and user3 should not see them (failed) (attempt 3)](https://github.com/user-attachments/assets/783aa5a3-f475-4ce8-8a7b-143c378c61e5)

Reply:
@Dzuming you meant 8.14.3 to 8.15.0? I will change it on the master branch
@lodyczekoladowe so, please take a look at the problem with this test in case of 8.x

Reply:
@Dzuming you meant 8.14.3 to 8.15.0? I will change it on the master branch
@lodyczekoladowe so, please take a look at the problem with this test in case of 8.x

Reply:
> you meant 8.14.3 to 8.15.0?

Yes, exactly